README.md

```diff
@@ -63,6 +63,40 @@ For more information about `bld2repo`, please see
 [bld2repo/README.md](bld2repo/README.md)
 
 
+## Installation instructions
+
+The `modulemd-tools` package is available in the official Fedora
+repositories, and RHEL 8.5 and higher. As such, it can be easily
+installed with:
+
+```
+dnf install modulemd-tools
+```
+
+There is also a Copr repository providing up-to-date stable builds for
+EPEL8. It is recommended to use this repository for installing
+`modulemd-tools` on RHEL 8.4 and lower.
+
+```
+dnf copr enable frostyx/modulemd-tools-epel
+dnf install modulemd-tools
+```
+
+If you prefer to install the latest stable package from this
```

Comment:
I guess this needs to be done after cloning the repo and running this from it's root folder...

Reply:
Good point. Please see the code block below, I added a cloning command there, so it is obvious.